### PR TITLE
Feat: support idleMinutes field in pipeline

### PR DESCRIPTION
**Pull request by zoumo**

### CODE REVIEW

**Comment by zoumo** [comment]
Can someone tell me what's wrong with my PR?

**Comment by carlossg** [comment]
it's in the linked jenkins report at https://jenkins.ci.cloudbees.com/job/plugins/job/kubernetes-plugin/542/console ?

```
[INFO] Error size is 0
[INFO] Total bugs: 1
[INFO] Self assignment of field PodTemplateStep.idleMinutes in org.csanchez.jenkins.plugins.kubernetes.pipeline.PodTemplateStep.setIdleMinutes(int) [org.csanchez.jenkins.plugins.kubernetes.pipeline.PodTemplateStep] At PodTemplateStep.java:[line 133] SA_FIELD_SELF_ASSIGNMENT
```

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java [hunk 1]
```diff
@@ -122,6 +124,15 @@ public void setInstanceCap(int instanceCap) {
         this.instanceCap = instanceCap;
     }
 
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    @DataBoundSetter
+    public void setIdleMinutes(int instanceCap) {
+        this.idleMinutes = idleMinutes;
```
Comment (iocanel) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java]: @zoumo: This is why the build fails. I guess you need to change the method argument to `idleMinutes`.

Comment (zoumo) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java]: thank you.

**Comment by zoumo** [comment]
got it

**Comment by zoumo** [comment]
@carlossg @iocanel help doc is ok now

**Comment by carlossg** [comment]
Thanks!

**Pull request merged**